packages/wonder-blocks-button/components/button.js

```diff
@@ -26,7 +24,7 @@ export type SharedProps = {
     /**
      * The color of the button, either blue or red.
      */
-    color: ValidTints,
+    color: "default" | "destructive",
```

Comment:
🤷‍♂️ Is "color" even the right term to use? intent? type? are we using color on mobile?

Reply:
I believe we're using `color` on mobile.  I'm open to changing it.